server/index.js

```diff
@@ -89,25 +89,112 @@ function getCurrentFixtures(round, currentSeason, leagueId) {
     return response.data.response;
   });
 }
+app.get('/api/team-form', (req, res, next) => {
+  const sql = `
+    select "matchDetails"
+    From "teamForm"
+    `;
+  db.query(sql)
+    .then(result => {
+      const dbresult = result.rows;
+      const flatten = dbresult.flat(1);
```

Comment:
If you always have to flatten it, why not just store it flattened in the first place?